person.rb

```diff
@@ -1,6 +1,7 @@
 require './nameable'
+require './rental'
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
 
   def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
Kindly add a method for adding rentals inside the person class in order to achieve the has_many association 👍👍😀